jjb/template.yaml

```diff
@@ -916,3 +916,29 @@
           timeout-var: 'BUILD_TIMEOUT'
           fail: true
           type: absolute
+
+- job-template:
+    name: 'FreeBSD-{branch}-{target_arch}-reproducibility-{testtype}'
+    defaults: global
+    node: jailer
+    scm:
+      - FreeBSD-src-use_git_commit
+    parameters:
+      - USE_GIT_COMMIT
+    builders:
+      - add-svn-revision-description
+      - checkout-scripts
+      - setup-jail
+      - execute-in-jail
+    publishers:
+      - scan-gcc-warnings
```

Comment:
I guess we don't really need to collect gcc warnings in most reproducibility jobs.